tests/map/test-map.c

```diff
@@ -0,0 +1,111 @@
+#include <assert.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+#include "map.h"
+
+void swap(int *x, int *y)
```

Comment:
Declare this internal function as a `static` one to avoid potential symbol conflicts.

Reply:
static inline ?

Reply:
No, simply use `static` since we don't care about further optimizations for a simple program.